matron/src/events.c

```diff
@@ -310,20 +310,15 @@ static void handle_event(union event_data *ev) {
     case EVENT_CUSTOM:
         w_handle_custom_weave(&(ev->custom));
         break;
-    case EVENT_SCREEN_RESULT_TEXT_EXTENTS:
-	    w_handle_screen_result_text_extents(&ev->screen_result_text_extents);
-	    break;
-    case EVENT_SCREEN_RESULT_CURRENT_POINT:
-	    w_handle_screen_result_current_point(&ev->screen_result_current_point);
-	    break;
-    case EVENT_SCREEN_RESULT_PEEK:
-	    w_handle_screen_result_peek(&ev->screen_result_peek);
-	    break;
-    /* case EVENT_SCREEN_RESULTS: */
-    /* 	w_handle_screen_results(&ev->screen_results); */
-    /* 	break; */
-	    w_handle_screen_result_peek(&ev->screen_result_peek);
-	    break;
+    // case EVENT_SCREEN_RESULT_TEXT_EXTENTS:
```

Comment:
Just for documentation, would be nice to have a comment explaining the context for the commented-out code. Is this left in because it will be uncommented with the future deprecation of the synchronous API?

Reply:
no, it's just dead. i guess i left as a reminder that we still have this akward situation where those three event types are defined as if they were main-loop events, but aren't actually handled in the main loop. for principle of least surprise i should define them in `screen_results` instead. (i'll make that change next session.)